modules/ROOT/pages/prereq-network.adoc

```diff
@@ -55,4 +48,20 @@ You must register the same SSL certificate on every server containing Mule Runti
 
 == SMTP Server Requirements
 
-Your network must include an SMTP server to manage e-mail alerts that are triggered by the platform.
\ No newline at end of file
+Your network must include an SMTP server to manage e-mail alerts that are triggered by the platform.
+
+== External Identity Provider and Client Management 
+
+To configure an Exernal Identity Provider for users single sign on (LDAP, OpenID, SAML) or for external client management as OpenID Connect, the cluster must be able to access the External Identity Provider endpoint. 
+
+== IP Forwarding and Bridging
```

Comment:
Probably I will move this one more time. It's difficult to categorize into Network or Software requirements since most software requirements are related with networking. Ideas welcomed.